foundation_cms/templates/patterns/components/footer.html

```diff
@@ -18,23 +18,20 @@
       </div>
       <div class="site-footer-ns__grid-cell site-footer-ns__links body-text-regular">
         <ul class="internal">
-          <li><a href="#">Meet Mozilla</a></li>
-          <li><a href="#">Leadership</a></li>
-          <li><a href="#">Public Records</a></li>
-          <li><a href="#">Licensing</a></li>
-          <li><a href="#">Press Center</a></li>
-          <li><a href="#">Contact us</a></li>
+          <li><a href="/en/meet-mozilla/website-licensing/">Licensing</a></li>
+          <li><a href="/en/meet-mozilla/annual-reports-and-financials/">Annual Reports &amp; Financials</a></li>
+          <li><a href="/en/meet-mozilla/press-center/">Press Center</a></li>
```

Comment:
```suggestion
          <li><a href="/meet-mozilla/website-licensing/">Licensing</a></li>
          <li><a href="/meet-mozilla/annual-reports-and-financials/">Annual Reports &amp; Financials</a></li>
          <li><a href="/meet-mozilla/press-center/">Press Center</a></li>
```

Reply:
Hi @ramram-mf! I don't think we need the /en/ in these urls. Our site will redirect them accordingly, also, lets say we were to redirect them to another locale, like es, since the page has not been translated yet, it will still show the English content anyways 👍 

I think with removing the /en/ from these links, we are reducing work needed in the future when we do introduce translated pages!